cpp/src/arrow/scalar_test.cc

```diff
@@ -1178,10 +1196,10 @@ class TestListLikeScalar : public ::testing::Test {
     CheckListCast(
         scalar, fixed_size_list(value_->type(), static_cast<int32_t>(value_->length())));
 
-    CheckInvalidListCast(scalar, fixed_size_list(value_->type(), 5),
-                         "Cannot cast " + scalar.type->ToString() + " of length " +
-                             std::to_string(value_->length()) +
-                             " to fixed size list of length 5");
+    auto invalidCastType = fixed_size_list(value_->type(), 5);
```

Comment:
Could you use `snake_case`?

```suggestion
    auto invalid_cast_type = fixed_size_list(value_->type(), 5);
```

Reply:
Fixed it.